src/neo/Cryptography/Helper.cs

```diff
@@ -109,6 +111,88 @@ public static byte[] Sha256(this Span<byte> value)
             return Sha256((ReadOnlySpan<byte>)value);
         }
 
+        public static byte[] AES256Decrypt(this byte[] data, byte[] key)
+        {
+            using Aes aes = Aes.Create();
+            aes.Key = key;
+            aes.Mode = CipherMode.ECB;
+            aes.Padding = PaddingMode.PKCS7;
+            using ICryptoTransform decryptor = aes.CreateDecryptor();
+            return decryptor.TransformFinalBlock(data, 0, data.Length);
+        }
+
+        public static byte[] AES256Encrypt(this byte[] data, byte[] key)
+        {
+            using Aes aes = Aes.Create();
+            aes.Key = key;
+            aes.Mode = CipherMode.ECB;
+            aes.Padding = PaddingMode.PKCS7;
+            using ICryptoTransform encryptor = aes.CreateEncryptor();
+            return encryptor.TransformFinalBlock(data, 0, data.Length);
+        }
+
+        public static byte[] ECEncrypt(byte[] message, ECPoint pubKey)
+        {
+            // P=kG,R=rG =>{R,M+rP}
+            BigInteger r, rx;
+            ECPoint R;
+            var curve = pubKey.Curve;
+            //r > N
+            using (RandomNumberGenerator rng = RandomNumberGenerator.Create())
+            {
+                do
+                {
+                    do
+                    {
+                        r = rng.NextBigInteger((int)curve.N.GetBitLength());
+                    }
+                    while (r.Sign == 0 || r.CompareTo(curve.N) >= 0);
+                    R = ECPoint.Multiply(curve.G, r);
+                    BigInteger x = R.X.Value;
+                    rx = x.Mod(curve.N);
+                }
+                while (rx.Sign == 0);
+            }
+            byte[] RBar = R.EncodePoint(true);
+            var z = ECPoint.Multiply(pubKey, r).X; // z = r * P = r* k * G
+            var Z = z.ToByteArray();
+            var EK = Z.Sha256();
+            var EM = message.AES256Encrypt(EK);
+            return RBar.Concat(EM).ToArray();
+        }
+        public static byte[] ECDecrypt(byte[] cypher, byte[] priKey, ECPoint pubKey)
```

Comment:
`KeyPair` instead of private and public?

Reply:
OK.I will change it.